src/test/java/systemtests/AddCommandSystemTest.java

```diff
@@ -73,27 +73,23 @@ public void add() {
         expectedResultMessage = RedoCommand.MESSAGE_SUCCESS;
         assertCommandSuccess(command, model, expectedResultMessage);
 
-        /* Case: add a entry with all fields same as another entry in the address book except name -> added */
-        toAdd = new EntryBuilder(AMY).withTitle(VALID_TITLE_BOB).build();
-        command = AddCommand.COMMAND_WORD + TITLE_DESC_BOB + DESCRIPTION_DESC_AMY + LINK_DESC_AMY + ADDRESS_DESC_AMY
+        /* Case: add a entry with all fields same as another entry in the entry book except link -> added */
+        toAdd = new EntryBuilder(AMY).withLink(VALID_LINK_BOB).build();
+        command = AddCommand.COMMAND_WORD + TITLE_DESC_AMY + DESCRIPTION_DESC_AMY + LINK_DESC_BOB + ADDRESS_DESC_AMY
                 + TAG_DESC_TECH;
         assertCommandSuccess(command, toAdd);
 
-        /* Case: add a entry with all fields same as another entry in the address book except phone and email
+        // Deprecated
+        /* Case: add a entry with all fields same as another entry in the entry book except description and link
          * -> added
          */
+        /*
         toAdd = new EntryBuilder(AMY).withDescription(VALID_DESCRIPTION_BOB).withLink(VALID_LINK_BOB).build();
         command = EntryUtil.getAddCommand(toAdd);
         assertCommandSuccess(command, toAdd);
+        */
 
-        /* Case: using alias to add a entry without tags to a non-empty address book,
-         * command with leading spaces and trailing spaces -> added
-         */
-        command = "   " + AddCommand.COMMAND_ALIAS + "  " + TITLE_DESC_BOB + "  " + DESCRIPTION_DESC_BOB + " "
-                + LINK_DESC_BOB + "   " + ADDRESS_DESC_BOB + "   " + TAG_DESC_TECH + "   " + TAG_DESC_SCIENCE + " ";
-        assertCommandSuccess(command, BOB);
```

Comment:
Why was this test removed?

Reply:
It was moved to https://github.com/CS2103-AY1819S2-W10-1/main/pull/61/files/e379232b05f7947b75cb6abd7449471bbe909d06..aa297d489957d10eed8b246ec5e7acc0c6ef82fd#diff-cf359788cb65ddab388e78c2f5ecdbf6R192